python/tests/Examples.py

```diff
@@ -568,6 +570,33 @@ def test_simple_hypothesis_with_importance_with_cluster(self):
 
         self.dump_graph(g, "Simple hypothesis with importance with cluster")
 
+    def create_a_simple_cluster_with_handle(self):
```

Comment:
Without "test" as the first word in the test function name, unit test will skip this function/test.